customer-service/channel-integration-framework/TwilioSampleInteg_version2/TwilioSampleInteg.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
why you removed 3.1?

Reply:
added back as it was not an intended change to change the target framework back to 2.1